src/main/scala/com/campudus/tableaux/database/domain/column.scala

```diff
@@ -1,37 +1,63 @@
 package com.campudus.tableaux.database.domain
 
 import com.campudus.tableaux.database.CheckValidValue._
-import com.campudus.tableaux.database._
 import com.campudus.tableaux.database.model.AttachmentFile
 import com.campudus.tableaux.database.model.TableauxModel._
+import com.campudus.tableaux.database.{LanguageNeutral, _}
 import org.joda.time.{DateTime, LocalDate}
 import org.vertx.scala.core.json._
 
-// TODO We should put all infos into a ColumnInfo case class or similar: Every change in a column needs a lot of changes elsewhere
+sealed trait ColumnInformation {
+  val table: Table
+  val id: ColumnId
+  val name: String
+  val ordering: Ordering
+  val identifier: Boolean
+  val displayInfos: Seq[DisplayInfo]
+}
+
+case class BasicColumnInformation(override val table: Table,
+                                  override val id: ColumnId,
+                                  override val name: String,
+                                  override val ordering: Ordering,
+                                  override val identifier: Boolean,
+                                  override val displayInfos: Seq[DisplayInfo]) extends ColumnInformation
+
+case class ConcatColumnInformation(override val table: Table) extends ColumnInformation {
+  override val name = "ID"
+
+  // Right now, every concat column is
+  // an identifier
+  override val identifier = true
+
+  override val id: ColumnId = 0
+  override val ordering: Ordering = 0
+  override val displayInfos: Seq[DisplayInfo] = List()
+}
+
 sealed trait ColumnType[+A] extends DomainObject {
-  val kind: TableauxDbType
 
-  val id: ColumnId
+  val kind: TableauxDbType
 
-  val name: String
+  val languageType: LanguageType
 
-  val table: Table
+  protected val columnInformation: ColumnInformation
 
-  val ordering: Ordering
+  final def table: Table = columnInformation.table
 
-  val languageType: LanguageType = SingleLanguage
+  final def id: ColumnId = columnInformation.id
 
-  val identifier: Boolean = false
+  final def name: String = columnInformation.name
 
-  val displayInfos: Seq[DisplayInfo]
+  final def ordering: Ordering = columnInformation.ordering
 
-  val countryCodes: Option[Seq[String]] = None
+  final def identifier: Boolean = columnInformation.identifier
 
   override def getJson: JsonObject = {
     // backward compatibility
     val multilanguage: Boolean = languageType match {
```

Comment:
Could be simplified to `val multilanguage: Boolean = !languageType.isInstanceOf[LanguageNeutral]`